lib/sanitizer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Sanitizer
+  def self.sanitize(str, decode = true)
```

Comment:
I think invocations of this method would be a lot clearer if `decode` was a keyword parameter. The default case would still be `Sanitizer.sanitize(foo)`, but the exceptional case would change from `Sanitizer.sanitze(foo, false)` to `Sanitizer.sanitze(foo, decode: false)`

Reply:
👍